src/operator/src/expr_helper.rs

```diff
@@ -295,9 +298,52 @@ pub fn validate_create_expr(create: &CreateTableExpr) -> Result<()> {
         }
     }
 
+    if auto_schema {
+        // `DateTime` type is not supported in auto schema mode.
```

Comment:
Can't we automatically convert it to `Timestamp(6)`?

Reply:
No, because it requires transferring data from `data_time_values` to `timestamp_values`, which is inefficient and inelegant.